src/cf-layout.less

```diff
@@ -1407,196 +1407,111 @@
-      margin-right: unit(-@grid_gutter-width / @base-font-size-px, em);
-      margin-left: 0;
-      padding-left: 0;
-      border-right-width: 0;
-      
-      // bleed background off left edge of page
-      &:before {
-        left: -200%;
-        background-color: @gray-5;
-        content: '';
-        display: block;
-        position: absolute;
-        width: 200%;
-        border-top: 1px solid @gray-50;
-        border-bottom: 1px solid @gray-50;
-        top: -1px;
-        bottom: -1px;
-      }
-    });
-  }
-  
-  
+
   &_text {
-    padding-top: unit(25px / @base-font-size-px, em); // approximates 30px visually
+    padding-top: unit(@grid_gutter-width / @base-font-size-px, em);
     padding-bottom: unit(@grid_gutter-width / @base-font-size-px, em);
-    
-    .respond-to-max(@mobile-max, {    
-      padding-right: unit((@grid_gutter-width /2) / @base-font-size-px, em);
-      padding-left: unit((@grid_gutter-width /2) / @base-font-size-px, em);
-    });
-    
-    .respond-to-range(@tablet-min, 799px, {    
+
+    .respond-to-min(@fcm-bp-min, {
+      width: 300px; // width of 5 of 12 cols at 799px - 30px padding
+
+      padding-right: unit(@grid_gutter-width / @base-font-size-px, em);
       padding-left: unit(@grid_gutter-width / @base-font-size-px, em);
     });
-    
-    .respond-to-min(@tablet-min, {
-      margin-right: unit((@grid_gutter-width /2) / @base-font-size-px, em);
-      width: 303px; // width of 5 of 12 cols at 799px - 30px padding
-      float: left;
-      border-width: 0;
-    });
-    
-    .respond-to-min(800px, {
+
+    .respond-to-min(@bp-med-min, {
       width: 270px; // width of 3 of 12 cols at 1140px - 15px padding
     });
-    
-  }
-  
-  &_tag {
-    .h4;
-    margin-bottom: unit(18px / @base-font-size-px, em); // approximates 30px visually
-  }
-  
-  &_icon {
-    .respond-to-max(@mobile-max, {
-      display: none;
-    });
-  }
-
-  &_headline {
-    .h2;
   }
 
-  &_body {
-    margin-bottom: unit(20px / @base-font-size-px, em);
-    .webfont-regular();
-    .respond-to-max(@mobile-max, {
-      .h4;
+  &_visual {
+    .respond-to-min(@fcm-bp-min, {
+      position: absolute;
+      left: 360px;
+      right: 0;
+      top: 0;
+      bottom: 0;
+      overflow: hidden;
     });
-  }
-
-  &_cta {
-    .webfont-regular();
-    
-    a& {
-      border-bottom-width: 1px;
-      cursor: pointer;
-    }
-  }
 
-  &_visual {
-    width: 100%;
-    line-height: 0;
-  }
- 
-  // crops image to center
-  &_wrapper {
-    text-align: center; 
-    overflow: hidden; 
+    .respond-to-min(@bp-med-min, {
+      left: 330px;
+    });
   }
 
   &_img {
-    .respond-to-min(@tablet-min, {
-      position: relative; 
+    display: block;
+
+    .respond-to-min(@fcm-bp-min, {
+      height: 100%;
+      max-width: initial;
+      position: relative;
       left: 50%;
-      transform: translate(-50%,0); 
-      max-width: 540px;
-      width: 540px;
+      transform: translateX(-50%);
     });
   }
```

Comment:
Do there need to be separate styles here for IE8, since it doesn't support transforms?